utils/training_utils.py

```diff
@@ -275,7 +280,11 @@ def evaluate(model, is_inception, loss_func, dataset_loader, test, mistakes_save
       # we ignore aux output in test loss calculation
       # since we aren't updating weights
       loss = loss_func(outputs, labels)
-      _, predictions = torch.max(outputs, 1)
+      batch_probabilities = probability_from_outputs(outputs)
```

Comment:
Can you just call the soft max here?

Like nn.Softmax(outputs, dim=1)?

Reply:
No, nn.Softmax returns a function